tests/unit/src/test/scala/tests/decorations/SyntheticDecorationsLspSuite.scala

```diff
@@ -297,28 +297,30 @@ class SyntheticDecorationsLspSuite extends BaseLspSuite("implicits") {
             |
             |/standalone/Main.scala
             |object Main{
-            |  "asd.".stripSuffix(".")
+            |  val value = "asd.".stripSuffix(".")
             |}
             |""".stripMargin
       )
       _ <- server.didChangeConfiguration(
         """|{
-           |  "show-implicit-conversions-and-classes": true
+           |  "show-implicit-conversions-and-classes": true,
+           |  "show-inferred-type": true
            |}
            |""".stripMargin
       )
       _ <- server.didOpen("standalone/Main.scala")
       _ = assertNoDiagnostics()
       _ = assertNoDiff(
+        // currently interactive semanticdb doesn't produce symbol signatures
         client.workspaceDecorations,
         """|object Main{
-           |  augmentString("asd.").stripSuffix(".")
+           |  val value = augmentString("asd.").stripSuffix(".")
```

Comment:
I discovered that symbol signatures are not produced by interactive semanticdb :/